### ISSUE TITLE
SVG morphing

### ISSUE THREAD

**Issue by DIYgod** [MoePlayer/DPlayer#134]
Dependence: https://github.com/alexk111/SVG-Morpheus

**Comment by donnes** [comment]
Your idea for this feature is build it in a separate bundle?
Ex.: dplayer-with-morpheus.min.js

I think make SVG animations optional is better for the final bundle size

**Comment by DIYgod** [comment]
No, I think we should regard `SVG-Morpheus` as a optional plugin. Load `svg-morpheus.js` before `DPlayer.min.js`, like [`hls.js`](http://dplayer.js.org/docs/#/?id=hls-support) and [`flv.js`](http://dplayer.js.org/docs/#/?id=flv-support).

**Comment by donnes** [comment]
Sure, I'll separate some time to analyze the impacts for this implantation

**Comment by hawkeye64** [comment]
Not worth it unless you want to maintain svg-morpheus. As you can see on the github link it is no longer maintained. https://github.com/alexk111/SVG-Morpheus

**Comment by DIYgod** [comment]
https://useanimations.com/#explore
